services/brig/src/Brig/Data/MLS/KeyPackage.hs

```diff
@@ -142,6 +144,22 @@ deleteKeyPackages u c suite refs =
     deleteQuery :: PrepQuery W (UserId, ClientId, CipherSuiteTag, [KeyPackageRef]) ()
     deleteQuery = "DELETE FROM mls_key_packages WHERE user = ? AND client = ? AND cipher_suite = ? AND ref in ?"
 
+deleteAllKeyPackages ::
+  (MonadClient m, MonadUnliftIO m) =>
+  UserId ->
+  ClientId ->
+  [CipherSuiteTag] ->
+  m ()
+deleteAllKeyPackages u c suites =
+  pooledForConcurrentlyN_ 16 suites $ \suite ->
```

Comment:
Why do we need this?

Reply:
We need to make multiple queries, because the ciphersuite is part of the partition key. This is what we usually do in such cases.

Reply:
Alright, thank you, have you somehow hit performance boundaries when not doing it like this or is it just for good measure?

Reply:
I'm not sure, I'm just doing it here to keep it consistent with all the other cases.